src/Symfony/Bundle/Resources/config/doctrine_orm.xml

```diff
@@ -152,7 +152,7 @@
             <argument type="service" id="api_platform.iri_converter" />
             <argument type="service" id="api_platform.property_accessor" />
             <argument type="service" id="logger" on-invalid="ignore" />
-            <argument key="$identifiersExtractor" type="service" id="api_platform.identifiers_extractor.cached" on-invalid="ignore" />
```

Comment:
this was correct

Reply:
see above, i just get  "Call to a member function getIdentifiersFromItem() on null" if if keep it nullable and that conf.

Reply:
check if its not null before calling the class, if its null just use the old way

Reply:
Is it a backwards compatibility problem? Would injecting it in a setter be ok then?

Reply:
I updated it that way, is it ok or am i missing something?